internal/auth/providers/azure_test.go

```diff
@@ -0,0 +1,511 @@
+package providers
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/buzzfeed/sso/internal/pkg/sessions"
+	"github.com/buzzfeed/sso/internal/pkg/testutil"
+
+	jose "gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+const MicrosoftTenantID = "9188040d-6c67-4c5b-b112-36a304b66dad"
```

Comment:
Probably doesn't matter, but we'd might as well make these package-private (`microsoftTenantID`, `testClientID`).